examples/fundmover/README.md

```diff
@@ -1,36 +1,53 @@
 # Fundmover Demo App
 
   App demonstrates the usage of TwoPC DM with Maria Database ,
-  BankAccount and Wallet class  extends the TwoPCExtResourceCohortPolicy DM and 
+  BankAccount and Wallet class extends the TwoPCExtResourceCohortPolicy DM and 
   Finance class extends the TwoPCCoordinatorPolicy DM. 
 
-#Setup 
-  Follow the Below step to build and configure the Maria DB
+#Setup  
+Follow the Below step to build and configure the Maria DB
 
 1. docker pull mariadb/server:latest
-2. docker run --name mariadbdcap -e MYSQL_ROOT_PASSWORD=mysecretpw -d mariadb/server:latest
-   In Place of mysecretpw specify the password to be used 
-3. docker start mariadbdcap
-4. sudo docker exec -it mariadbdcap bash 
-5. mysql -uroot -p
-   use the password defined  in Step 2 
-6. Execute all  the query  present in sql-scripts/setup.sql file 
-7. Exit from the CLi and container 
+2. docker run --name mariadbamino -e MYSQL_ROOT_PASSWORD=mysecretpw -d mariadb/server:latest  
+In Place of mysecretpw specify the password to be used 
+3. docker start mariadbamino
+4. sudo docker exec -it mariadbamino bash 
+5. mysql -uroot -p  
```

Comment:
suggestion: trail space

Reply:
Trailing space is used in markdown file for adding a newline.
https://stackoverflow.com/questions/33191744/how-to-add-new-line-in-markdown-presentation